src/main/java/frc/robot/commands/DriveCommand.java

```diff
@@ -0,0 +1,54 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018-2019 FIRST. All Rights Reserved.                        */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+package frc.robot.commands;
+
+import frc.robot.subsystems.ConditionedDriveSubsystem;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+
+import frc.robot.controllers.ControllerBase;
+
+public class DriveCommand extends CommandBase {
+  private final ControllerBase m_controller;
+  private final ConditionedDriveSubsystem m_cdss;
+
+  public DriveCommand(ConditionedDriveSubsystem cdss, ControllerBase controller) {
+    m_controller = controller;
+    m_cdss = cdss;
+    // Use addRequirements() here to declare subsystem dependencies.
+    addRequirements(cdss);
+  }
+
+  // Called when the command is initially scheduled.
+  @Override
+  public void initialize() {
+    stop();
+  }
+
+  // Called every time the scheduler runs while the command is scheduled.
+  @Override
+  public void execute() {
+    m_cdss.mapToLimit(m_controller.getMoveRequest(), m_controller.getTurnRequest());
```

Comment:
Maybe instead of having the conditioner hold a reference to the DriveSubsystem (which I feel somewhat breaks the idea of the command-subsystem structure), you could get the conditioned value from the conditioner then feed it to the DriveSubsystem?
```
double conditioned_mr = conditioner.conditionMoveRequest(m_controller.getMoveRequest());
double conditioned_tr = conditioner.conditionTurnRequest(m_controller.getTurnRequest());
m_drive_subsystem.drive(conditioned_mr, conditioned_tr);
```